omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/LocationAttributeController1_9Test.java

```diff
@@ -94,4 +99,26 @@ public void shouldVoidAttribute() throws Exception {
 		Assert.assertTrue(locationAttribute.isVoided());
 		Assert.assertEquals("unit test", locationAttribute.getVoidReason());
 	}
+
+	@Test
+	public void shouldReturnOnlyAttributesOfGivenTypeWhenSearch() throws Exception {
+		String attributeTypeUuid = "9516cc50-6f9f-11e0-8414-001e378eb67e";
+		searchAttributeAndCheck(attributeTypeUuid);
+
+		String anotherAttributeTypUuid = "9516cc50-6f9f-132r-6556-001e378eb67f";
+		searchAttributeAndCheck(anotherAttributeTypUuid);
+	}
+
+	/**
+	 * sends search request for attributes with given type, checks if all result attributes are of this type
+	 * @param attributeTypeUuid
+	 * @throws Exception
+     */
+	private void searchAttributeAndCheck(String attributeTypeUuid) throws Exception {
+		SimpleObject response2 = deserialize(handle(newGetRequest(getURI(), new Parameter("attributeType", attributeTypeUuid))));

```

Comment:
Looks like this test is looking up an attribute by uuid and not really searching